en/docs/adv_examples/resnet.md

```diff
@@ -1,373 +1,353 @@
-## 简介 Introduction
+## Introduction
 
-### 图像分类与CNN
+### Image classification and CNN
 
- **图像分类** 是指将图像信息中所反映的不同特征，把不同类别的目标区分开来的图像处理方法，是计算机视觉中其他任务，比如目标检测、语义分割、人脸识别等高层视觉任务的基础。
+**Image classification** is an image processing method that divided different features reflected in image information into different categories of targets. It is the basis of  other tasks in computer vision, such as detection, semantic segmentation, face recognition and other high-level visual tasks. 
```

Comment:
is an image processing method that divided different features reflected in image information into different categories of targets

->
is an image processing method that divides objects into different categories from the different features reflected in image information.